AX: Add AccessibilityObjectInlines.h to guarantee the inlining of hot functions like AccessibilityObject::axObjectCache()

twilco opened the pull request:
#### abbdd705cb616d8c06a93b1efe784d8c7766a14a
<pre>
AX: Add AccessibilityObjectInlines.h to guarantee the inlining of hot functions like AccessibilityObject::axObjectCache()
<a href="https://bugs.webkit.org/show_bug.cgi?id=298162">https://bugs.webkit.org/show_bug.cgi?id=298162</a>
<a href="https://rdar.apple.com/159531037">rdar://159531037</a>

Reviewed by Joshua Hoffman.

Inline various small and frequently called functions.

* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/accessibility/AXObjectCache.cpp:
* Source/WebCore/accessibility/AccessibilityImageMapLink.cpp:
* Source/WebCore/accessibility/AccessibilityMathMLElement.cpp:
* Source/WebCore/accessibility/AccessibilityNodeObject.cpp:
* Source/WebCore/accessibility/AccessibilityObject.cpp:
(WebCore::AccessibilityObject::init): Deleted.
(WebCore::AccessibilityObject::axObjectCache const): Deleted.
(WebCore::AccessibilityObject::treeID const): Deleted.
(WebCore::AccessibilityObject::isDetached const): Deleted.
(WebCore::AccessibilityObject::initializeAncestorFlags): Deleted.
(WebCore::AccessibilityObject::accessibleNameDerivesFromHeading const): Deleted.
(WebCore::AccessibilityObject::isARIATextControl const): Deleted.
(WebCore::AccessibilityObject::isNonNativeTextControl const): Deleted.
(WebCore::AccessibilityObject::textMarkerRange const): Deleted.
(WebCore::AccessibilityObject::frame const): Deleted.
(WebCore::AccessibilityObject::hasRowGroupTag const): Deleted.
(WebCore::AccessibilityObject::supportsCheckedState const): Deleted.
(WebCore::AccessibilityObject::supportsAutoComplete const): Deleted.
(WebCore::AccessibilityObject::protectedDocument const): Deleted.
(WebCore::AccessibilityObject::children): Deleted.
(WebCore::AccessibilityObject::ariaIsMultiline const): Deleted.
(WebCore::AccessibilityObject::supportsCurrent const): Deleted.
(WebCore::AccessibilityObject::hasElementName const): Deleted.
(WebCore::AccessibilityObject::integralAttribute const): Deleted.
(WebCore::AccessibilityObject::colorValue const): Deleted.
(WebCore::AccessibilityObject::datetimeAttributeValue const): Deleted.
(WebCore::AccessibilityObject::linkRelValue const): Deleted.
(WebCore::AccessibilityObject::isInlineText const): Deleted.
(WebCore::AccessibilityObject::supportsKeyShortcuts const): Deleted.
(WebCore::AccessibilityObject::keyShortcuts const): Deleted.
(WebCore::AccessibilityObject::element const): Deleted.
(WebCore::AccessibilityObject::commandType const): Deleted.
(WebCore::AccessibilityObject::hasDatalist const): Deleted.
(WebCore::AccessibilityObject::supportsChecked const): Deleted.
(WebCore::AccessibilityObject::supportsRowCountChange const): Deleted.
(WebCore::AccessibilityObject::setLastKnownIsIgnoredValue): Deleted.
(WebCore::AccessibilityObject::ignoredFromPresentationalRole const): Deleted.
(WebCore::AccessibilityObject::textIteratorIgnoringFullSizeKana): Deleted.
(WebCore::AccessibilityObject::isIgnoredByDefault const): Deleted.
(WebCore::AccessibilityObject::isRenderHidden const): Deleted.
(WebCore::AccessibilityObject::elementName const): Deleted.
(WebCore::AccessibilityObject::isFigureElement const): Deleted.
(WebCore::AccessibilityObject::isOutput const): Deleted.
(WebCore::AccessibilityObject::ensureRareData): Deleted.
* Source/WebCore/accessibility/AccessibilityObjectInlines.h: Added.
(WebCore::AccessibilityObject::init):
(WebCore::AccessibilityObject::axObjectCache const):
(WebCore::AccessibilityObject::isDetached const):
(WebCore::AccessibilityObject::isNonNativeTextControl const):
(WebCore::AccessibilityObject::textMarkerRange const):
(WebCore::AccessibilityObject::frame const):
(WebCore::AccessibilityObject::hasRowGroupTag const):
(WebCore::AccessibilityObject::hasElementName const):
(WebCore::AccessibilityObject::protectedDocument const):
(WebCore::AccessibilityObject::colorValue const):
(WebCore::AccessibilityObject::isInlineText const):
(WebCore::AccessibilityObject::element const):
(WebCore::AccessibilityObject::commandType const):
(WebCore::AccessibilityObject::hasDatalist const):
(WebCore::AccessibilityObject::textIteratorIgnoringFullSizeKana):
(WebCore::AccessibilityObject::isIgnoredByDefault const):
(WebCore::AccessibilityObject::isRenderHidden const):
(WebCore::AccessibilityObject::elementName const):
(WebCore::AccessibilityObject::isFigureElement const):
(WebCore::AccessibilityObject::isOutput const):
(WebCore::AccessibilityObject::ensureRareData):
(WebCore::AccessibilityObject::setLastKnownIsIgnoredValue):
(WebCore::AccessibilityObject::ignoredFromPresentationalRole const):
(WebCore::AccessibilityObject::scrollToMakeVisible const):
(WebCore::AccessibilityObject::supportsChecked const):
(WebCore::AccessibilityObject::supportsRowCountChange const):
(WebCore::AccessibilityObject::datetimeAttributeValue const):
(WebCore::AccessibilityObject::linkRelValue const):
(WebCore::AccessibilityObject::supportsKeyShortcuts const):
(WebCore::AccessibilityObject::keyShortcuts const):
(WebCore::AccessibilityObject::integralAttribute const):
(WebCore::AccessibilityObject::supportsCurrent const):
(WebCore::AccessibilityObject::ariaIsMultiline const):
(WebCore::AccessibilityObject::children):
(WebCore::AccessibilityObject::supportsCheckedState const):
(WebCore::AccessibilityObject::supportsAutoComplete const):
(WebCore::AccessibilityObject::isARIATextControl const):
(WebCore::AccessibilityObject::accessibleNameDerivesFromHeading const):
(WebCore::AccessibilityObject::initializeAncestorFlags):
(WebCore::AccessibilityObject::treeID const):
* Source/WebCore/accessibility/AccessibilityTableCell.cpp:
* Source/WebCore/accessibility/cocoa/AXCoreObjectCocoa.mm:
* Source/WebCore/accessibility/cocoa/AccessibilityObjectCocoa.mm:
* Source/WebCore/accessibility/ios/AccessibilityObjectIOS.mm:
* Source/WebCore/accessibility/isolatedtree/AXIsolatedTree.cpp:
* Source/WebCore/accessibility/mac/AccessibilityObjectMac.mm:
* Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperMac.mm:
* Source/WebCore/inspector/InspectorAuditAccessibilityObject.cpp:
* Source/WebCore/inspector/agents/InspectorDOMAgent.cpp:

Canonical link: <a href="https://commits.webkit.org/299468@main">https://commits.webkit.org/299468@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/5a3af920e39f78c051bf00da639d527efc3abd9b

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/119049 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/38730 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/29381 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/125256 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/71112 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/da2541a1-3ce1-41af-bedd-fe9d2f41ab7d) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/120927 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/39426 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/47312 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/90393 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/59840 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/3183f29d-e720-40de-bc17-1365ad46bf2c) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/122002 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/31437 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/106728 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/70853 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/afeb2495-3065-4841-b08d-31201cc81172) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/30499 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/24844 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/68912 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/100884 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/25030 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/128290 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/45956 "Built successfully") | [❌ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/34727 "Found 1 new test failure: imported/w3c/web-platform-tests/html/anonymous-iframe/local-storage.tentative.https.window.html (failure)") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/99014 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/46323 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/102948 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/98793 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/25123 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/44250 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/22251 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/42535 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/45826 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/51505 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/45293 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/48638 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/46978 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/64e95cd66131ea50b191cb570ec84765bab31562. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50110

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/4eca14a82c1a8b1d541523b58b9b3c8c6a525570. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50110

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/f8fc0d2aea695ead3ff7d4358e1947e4d92f6955. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50110

webkit-early-warning-system commented:
EWS run on current version of this PR (hash https://github.com/WebKit/WebKit/commit/641e0c8cb270d8d78df8ba81937b59b0a1d3b6ff)<details>

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/118786 "Build is in progress. Recent messages:") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/38467 "Build is in progress. Recent messages:Running configuration") | [⏳ 🛠 mac ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 wpe ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 win ](https://ews-build.webkit.org/#/builders/Win-Build-EWS "Waiting in queue, processing has not started yet") | ⏳ 🛠 ios-apple 
| [⏳ 🧪 bindings ](https://ews-build.webkit.org/#/builders/Bindings-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 ios-sim ](https://ews-build.webkit.org/#/builders/iOS-18-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 mac-AS-debug ](https://ews-build.webkit.org/#/builders/macOS-Sequoia-Debug-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 wpe-wk2 ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 win-tests ](https://ews-build.webkit.org/#/builders/Win-Build-EWS "Waiting in queue, processing has not started yet") | ⏳ 🛠 mac-apple 
| [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/121739 "Build is in progress. Recent messages:") | [⏳ 🧪 ios-wk2 ](https://ews-build.webkit.org/#/builders/iOS-18-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-mac ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-wpe ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | | ⏳ 🛠 vision-apple 
| | [⏳ 🧪 ios-wk2-wpt ](https://ews-build.webkit.org/#/builders/iOS-18-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-wk1 ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 wpe-cairo ](https://ews-build.webkit.org/#/builders/WPE-Cairo-Build-EWS "Waiting in queue, processing has not started yet") | | 
| [⏳ 🛠 🧪 jsc ](https://ews-build.webkit.org/#/builders/JSC-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-ios ](https://ews-build.webkit.org/#/builders/iOS-18-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 gtk ](https://ews-build.webkit.org/#/builders/GTK-Build-EWS "Waiting in queue, processing has not started yet") | | 
| [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/117300 "Build is in progress. Recent messages:") | [⏳ 🛠 vision ](https://ews-build.webkit.org/#/builders/visionOS-2-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-AS-debug-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Sequoia-Debug-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 gtk-wk2 ](https://ews-build.webkit.org/#/builders/GTK-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/46058 "Build is in progress. Recent messages:") | [⏳ 🧪 mac-wk2-stress ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-gtk ](https://ews-build.webkit.org/#/builders/GTK-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [⏳ 🧪 vision-wk2 ](https://ews-build.webkit.org/#/builders/visionOS-2-Simulator-WK2-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-intel-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 playstation ](https://ews-build.webkit.org/#/builders/PlayStation-Build-EWS "Waiting in queue, processing has not started yet") | | 
| | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/45563 "Build is in progress. Recent messages:Running configuration") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/51241 "Build is in progress. Recent messages:") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/145996 "Build is in progress. Recent messages:") | | 
| | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/45027 "Build is in progress. Recent messages:Running configuration") | | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/35/builds/145996 "Build is in progress. Recent messages:") | | 
| | [⏳ 🛠 watch ](https://ews-build.webkit.org/#/builders/watchOS-11-Build-EWS "Waiting in queue, processing has not started yet") | | | | 
| | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/46713 "Build is in progress. Recent messages:") | | | | </details>

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/5a3af920e39f78c051bf00da639d527efc3abd9b. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50110

twilco committed:
AX: Add AccessibilityObjectInlines.h to guarantee the inlining of hot functions like AccessibilityObject::axObjectCache()
https://bugs.webkit.org/show_bug.cgi?id=298162
rdar://159531037

Reviewed by Joshua Hoffman.

Inline various small and frequently called functions.

* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/accessibility/AXObjectCache.cpp:
* Source/WebCore/accessibility/AccessibilityImageMapLink.cpp:
* Source/WebCore/accessibility/AccessibilityMathMLElement.cpp:
* Source/WebCore/accessibility/AccessibilityNodeObject.cpp:
* Source/WebCore/accessibility/AccessibilityObject.cpp:
(WebCore::AccessibilityObject::init): Deleted.
(WebCore::AccessibilityObject::axObjectCache const): Deleted.
(WebCore::AccessibilityObject::treeID const): Deleted.
(WebCore::AccessibilityObject::isDetached const): Deleted.
(WebCore::AccessibilityObject::initializeAncestorFlags): Deleted.
(WebCore::AccessibilityObject::accessibleNameDerivesFromHeading const): Deleted.
(WebCore::AccessibilityObject::isARIATextControl const): Deleted.
(WebCore::AccessibilityObject::isNonNativeTextControl const): Deleted.
(WebCore::AccessibilityObject::textMarkerRange const): Deleted.
(WebCore::AccessibilityObject::frame const): Deleted.
(WebCore::AccessibilityObject::hasRowGroupTag const): Deleted.
(WebCore::AccessibilityObject::supportsCheckedState const): Deleted.
(WebCore::AccessibilityObject::supportsAutoComplete const): Deleted.
(WebCore::AccessibilityObject::protectedDocument const): Deleted.
(WebCore::AccessibilityObject::children): Deleted.
(WebCore::AccessibilityObject::ariaIsMultiline const): Deleted.
(WebCore::AccessibilityObject::supportsCurrent const): Deleted.
(WebCore::AccessibilityObject::hasElementName const): Deleted.
(WebCore::AccessibilityObject::integralAttribute const): Deleted.
(WebCore::AccessibilityObject::colorValue const): Deleted.
(WebCore::AccessibilityObject::datetimeAttributeValue const): Deleted.
(WebCore::AccessibilityObject::linkRelValue const): Deleted.
(WebCore::AccessibilityObject::isInlineText const): Deleted.
(WebCore::AccessibilityObject::supportsKeyShortcuts const): Deleted.
(WebCore::AccessibilityObject::keyShortcuts const): Deleted.
(WebCore::AccessibilityObject::element const): Deleted.
(WebCore::AccessibilityObject::commandType const): Deleted.
(WebCore::AccessibilityObject::hasDatalist const): Deleted.
(WebCore::AccessibilityObject::supportsChecked const): Deleted.
(WebCore::AccessibilityObject::supportsRowCountChange const): Deleted.
(WebCore::AccessibilityObject::setLastKnownIsIgnoredValue): Deleted.
(WebCore::AccessibilityObject::ignoredFromPresentationalRole const): Deleted.
(WebCore::AccessibilityObject::textIteratorIgnoringFullSizeKana): Deleted.
(WebCore::AccessibilityObject::isIgnoredByDefault const): Deleted.
(WebCore::AccessibilityObject::isRenderHidden const): Deleted.
(WebCore::AccessibilityObject::elementName const): Deleted.
(WebCore::AccessibilityObject::isFigureElement const): Deleted.
(WebCore::AccessibilityObject::isOutput const): Deleted.
(WebCore::AccessibilityObject::ensureRareData): Deleted.
* Source/WebCore/accessibility/AccessibilityObjectInlines.h: Added.
(WebCore::AccessibilityObject::init):
(WebCore::AccessibilityObject::axObjectCache const):
(WebCore::AccessibilityObject::isDetached const):
(WebCore::AccessibilityObject::isNonNativeTextControl const):
(WebCore::AccessibilityObject::textMarkerRange const):
(WebCore::AccessibilityObject::frame const):
(WebCore::AccessibilityObject::hasRowGroupTag const):
(WebCore::AccessibilityObject::hasElementName const):
(WebCore::AccessibilityObject::protectedDocument const):
(WebCore::AccessibilityObject::colorValue const):
(WebCore::AccessibilityObject::isInlineText const):
(WebCore::AccessibilityObject::element const):
(WebCore::AccessibilityObject::commandType const):
(WebCore::AccessibilityObject::hasDatalist const):
(WebCore::AccessibilityObject::textIteratorIgnoringFullSizeKana):
(WebCore::AccessibilityObject::isIgnoredByDefault const):
(WebCore::AccessibilityObject::isRenderHidden const):
(WebCore::AccessibilityObject::elementName const):
(WebCore::AccessibilityObject::isFigureElement const):
(WebCore::AccessibilityObject::isOutput const):
(WebCore::AccessibilityObject::ensureRareData):
(WebCore::AccessibilityObject::setLastKnownIsIgnoredValue):
(WebCore::AccessibilityObject::ignoredFromPresentationalRole const):
(WebCore::AccessibilityObject::scrollToMakeVisible const):
(WebCore::AccessibilityObject::supportsChecked const):
(WebCore::AccessibilityObject::supportsRowCountChange const):
(WebCore::AccessibilityObject::datetimeAttributeValue const):
(WebCore::AccessibilityObject::linkRelValue const):
(WebCore::AccessibilityObject::supportsKeyShortcuts const):
(WebCore::AccessibilityObject::keyShortcuts const):
(WebCore::AccessibilityObject::integralAttribute const):
(WebCore::AccessibilityObject::supportsCurrent const):
(WebCore::AccessibilityObject::ariaIsMultiline const):
(WebCore::AccessibilityObject::children):
(WebCore::AccessibilityObject::supportsCheckedState const):
(WebCore::AccessibilityObject::supportsAutoComplete const):
(WebCore::AccessibilityObject::isARIATextControl const):
(WebCore::AccessibilityObject::accessibleNameDerivesFromHeading const):
(WebCore::AccessibilityObject::initializeAncestorFlags):
(WebCore::AccessibilityObject::treeID const):
* Source/WebCore/accessibility/AccessibilityTableCell.cpp:
* Source/WebCore/accessibility/cocoa/AXCoreObjectCocoa.mm:
* Source/WebCore/accessibility/cocoa/AccessibilityObjectCocoa.mm:
* Source/WebCore/accessibility/ios/AccessibilityObjectIOS.mm:
* Source/WebCore/accessibility/isolatedtree/AXIsolatedTree.cpp:
* Source/WebCore/accessibility/mac/AccessibilityObjectMac.mm:
* Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperMac.mm:
* Source/WebCore/inspector/InspectorAuditAccessibilityObject.cpp:
* Source/WebCore/inspector/agents/InspectorDOMAgent.cpp:

Canonical link: https://commits.webkit.org/299468@main
